### PR TITLE
Remove Top Row, Add Pop Link Text field

### PR DESCRIPTION
**Pull request by RobLoweMOJ**
Created a duplicate on current acf fields thats not active  (backup of top row fields)
Removed top row acf fields
Removed top row code from agency single template
Added New field to set popup find out more text

### CODE REVIEW

File: web/app/themes/justicejobs/single-agency.php [hunk 1]
```diff
@@ -67,145 +67,6 @@ class="hero__img hero__img--mobile"
...
-                                            <?php endif ?>
-                                        </li>
-                                    <?php endwhile; ?>
-                                </ul>
-                            </div>
-                        <?php endif; ?>
-                    </div>
-                    <!-- / END TOP RIGHT CAROUSEL -->
-                <?php endif ?>
-
```
Comment (wilson1000-MoJ) [inline web/app/themes/justicejobs/single-agency.php:208]: There was a lot of text deleted from this file. What was the desired affect and has functionality been taken away? 
Thank you

Comment (RobLoweMOJ) [reply web/app/themes/justicejobs/single-agency.php:208]: The top row of the agency page blocks currently is broken. It requires a video. Josh has not used it due to this. We have decided as quick fix for content handover d to remove the top row section. With a plan to review the agency blocks in the future so they are more flexible. I removed the code as we can always get it back in GIT. But I can comment it out if you like.

Comment (RobLoweMOJ) [reply web/app/themes/justicejobs/single-agency.php:208]: So to answer your question yes we are taking away functionality for now. As its not used and will complicate content training handover.

Comment (wilson1000-MoJ) [reply web/app/themes/justicejobs/single-agency.php:208]: Great, thanks for the explanation :)

**Pull request merged**